tests/conftest.py

```diff
@@ -58,13 +58,14 @@ def lora_adapter_path(request: pytest.FixtureRequest) -> str:
 
 
 @pytest.fixture(
-    params=[
+    # params=[
         # pytest.param(True, id="disable-frontend-multiprocessing=True"),
-        pytest.param(False, id="disable-frontend-multiprocessing=False"),
-    ]
+        # pytest.param(False, id="disable-frontend-multiprocessing=False"),
+    # ]
 )
 def disable_frontend_multiprocessing(request):
     """Enable or disable the frontend-multiprocessing feature."""
+    return False
```

Comment:
Why were the parameters commented and the fixture modified to return an hardcoded `False`?

Reply:
my bad fixed in new commit